android/Useful_links.md

```diff
@@ -8,6 +8,7 @@
 
 ### Debug
 - [Stetho](http://facebook.github.io/stetho/#integrations) - A debug bridge for Android applications
+- [Log Util](https://github.com/JakeWharton/hugo) - Annotation-triggered method call logging for your debug builds

```

Comment:
Instead of "Log util", title should be "Hugo". I mean title should be a library name!